src/diffusers/models/vae_flax.py

```diff
@@ -600,7 +600,8 @@ def __call__(self, sample, sample_posterior=False, deterministic: bool = True, r
             hidden_states = posterior.latent_dist.sample(rng)
         else:
             hidden_states = posterior.latent_dist.mode()
-        hidden_states = self.decode(hidden_states, return_dict=return_dict).sample
+
+        sample = self.decode(hidden_states, return_dict=return_dict).sample
```

Comment:
wrong naming here led to interestingly only a difference of 0.06 -> corrected now